demos/calendar_starter/tasks.py

```diff
@@ -0,0 +1,65 @@
+import pickle 
+import os.path
```

Comment:
For now this will do, but next time try to create a seperate demo file that imports a copy of your task file